rust_snuba/src/consumer.rs

```diff
@@ -92,7 +92,7 @@ pub fn consumer_impl(
         configure_metrics(Box::new(StatsDBackend::new(
             &host,
             port,
-            "snuba.rust_consumer",
+            "snuba.consumer",
```

Comment:
How about swapping this out only if `--skip-write` is False (implying it is the primary consumer actually writing to clickhouse) and keeping the other prefix if not.

Otherwise are are definitely going to ruin our existing metrics/dashboards/monitors

Reply:
I can't reuse our existing dashboard widgets to validate when the prefix is not the same

I think our dashboard widgets _should_ always filter by consumer group as well, I haven't checked everything but I was planning to fix it that way.

Reply:
Ok. Keep in mind that our main consumer receive latency alert/SLO does not though. Are you planning to change that too? It's true that we're not emitting this particular metric from Rust right now but I expect this to change really soon.

Reply:
I didn't know that.. Ok, let me see if i can fix it tomorrow. If you think those are the only ones it seems manageable, I was definitely gonna just roll this out and see what breaks :D

Reply:
Now i wonder if any python reference consumers we are running also causes this to be wrong 🤔

Reply:
I checked and our main receive latency SLO _does_ filter by consumer group. I think I may have added it when it went off because of the python reference consumer deployment, and then forgot about it.